charts/aws-ebs-csi-driver/templates/controller.yaml

```diff
@@ -238,6 +238,7 @@ spec:
           image: {{ printf "%s:%s" .Values.sidecars.livenessProbeImage.repository .Values.sidecars.livenessProbeImage.tag }}
           args:
             - --csi-address=/csi/csi.sock
+            - --v={{ .Values.controller.logLevel }}
```

Comment:
I don't think that the liveness probe supports a `--v` flag

Reply:
```
➜  aws-ebs-csi-driver git:(master) docker run --rm -it k8s.gcr.io/sig-storage/livenessprobe:v2.2.0 -h                                                           <aws:saml>
Usage of /livenessprobe:
...
  -v value
    	number for the log level verbosity
...
```